.github/workflows/docker-image.yml

```
@@ -88,7 +88,7 @@
     - name: Update database
       working-directory: ./PerfumeTracker.Server
       env:
-       ConnectionStrings__DefaultConnection: ${{ secrets.POSTGRES_CONNSTR }}
+       ConnectionStrings__PerfumeTracker: ${{ secrets.POSTGRES_CONNSTR }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 7

(indentation)